CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ All notable, unreleased changes to this project will be documented in this file.
 # Unreleased
 
 ### Breaking changes
+- Convert IDs from DB to Graphql format in Email Payload - #9388 by @L3str4nge
```

Comment:
It is not exacly Email payload. It is more like generic payload for any type of notification. You can have an app that sends a text messages, or sends notification on any type of comunicator.

Reply:
(I see that task has `Email Payload` but maybe it would be good to rename it to something more generic, as it has an impact on all email plugins, and Notify webhook)